test/unit/v35.test.js

```diff
@@ -69,7 +69,7 @@ describe('v5', () => {
     assert.equal(v3('hello.example.com', v3.DNS), '9125a8dc-52ee-365b-a5aa-81b0b3681cf6');
     assert.equal(v3('http://example.com/hello', v3.URL), 'c6235813-3ba4-3801-ae84-e0a6ebb7d138');
     assert.equal(
-      v3('hello', '0f5abcd1-c194-47f3-905b-2df7263a084b'),
+      v3('hello', '0f5abcd1-c194-47f3-905b-2df7263a084b'.toUpperCase()),
```

Comment:
This would be a good opportunity to break up the huge `v3`/`v5` tests into several smaller `test()` cases, Ideally 1 or 2 assertions per test block.

Then I believe that the `toUpperCase()` test would deserve its own test case with a reasonable description like ("accepts uppercase uuid notation for namespace")

Reply:
I also believe we should, just for the sake of consistency, keep the `v3` and `v5` tests in sync and test the same stuff on both variants of the algorithm.